### ISSUE TITLE
A question about MNO

### ISSUE THREAD

**Issue by yoga-Yuki** [neuraloperator/markov_neural_operator#1]
Hello！I have a question about MNO. In calculating the process of dissipative loss，I found that this code is missing“x_diss = torch.tensor(sampling_fn(x.shape[0], radii), dtype=torch.float).to(device)”， The function “sample_uniform_spherical_shell” requires three input parameters, but only two parameters were entered in the code, resulting in an error during operation. So I have a question about the third parameter “shape”, what does this parameter refer to？

**Comment by mliuschi** [comment]
@yoga-Yuki Thanks for bringing this up! This was a misalignment between two different versions of the code that I forgot to resolve. Commit [67e77ed](https://github.com/neuraloperator/markov_neural_operator/commit/67e77ed04f3909b71bb7654be2944b353bcaf227) should resolve this issue. Please let me know if there are any other issues.

**Comment by yoga-Yuki** [comment]
Thank you for your reply！I have another question, there seems to be a missing function file (@chebfun) in the 'data_generation' folder regarding the generation of KS data.

**Comment by mliuschi** [comment]
Hi @yoga-Yuki, thanks for bringing this up and for your patience. We couldn't find the exact file used to generate the data, but [here's a slightly different Python implementation](https://drive.google.com/file/d/1WRm4PIW3U678lM4p8ZIpGOf9IanCnH3S/view?usp=sharing) written by @Zhengyu-Huang based on the same numerical method.
